feature-libs/product-configurator/rulebased/styles/_configurator-overview-form.scss

```diff
@@ -15,127 +15,319 @@
     padding-inline-end: 20px;
   }
 
-  .cx-group {
```

Comment:
There is a number of styling changes. 
Is there any task to cover them, for instance see `css-changes-in-version-5.md`?